### ISSUE TITLE
aws_lb_target_group requires port, vpc_id and protocol when target_type is lambda

### ISSUE THREAD

**Issue by daniilyar** [terraform-providers/terraform-provider-aws#7936]
### Terraform Version

0.11.11

### Affected Resource(s)

aws_lb_target_group

### Terraform Configuration Files

```hcl
resource "aws_lb_target_group" "test" {
  target_type = "lambda"
}
```

### Expected Behavior

aws_lb_target_group should be created

### Actual Behavior

```
Error: aws_lb_target_group.test: "port": required field is not set
Error: aws_lb_target_group.test: "protocol": required field is not set
Error: aws_lb_target_group.test: "vpc_id": required field is not set
```

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`

### Additional context

Documentation states that these are not required when target_type = "lambda"
AWS API also does not allow these to be set when target_type = "lambda":

```
ValidationError: Port cannot be specified for target groups with target type 'lambda'
```

**Comment by daniilyar** [comment]
This is a moved https://github.com/hashicorp/terraform/issues/20332 issue. It was moved because it seems that it is related to the AWS provider and not the core Terraform part

**Comment by burtsdenis** [comment]
@daniilyar, as I see, this bug reproduces on version 0.12.

**Comment by bflad** [comment]
Hi @daniilyar 👋 Which Terraform AWS Provider version are you using? e.g.

```
$ terraform providers
.
└── provider.aws 2.1.0
```

Support for `target_type = "lambda"` (along with validating values in that argument) was added via https://github.com/terraform-providers/terraform-provider-aws/issues/6719 and released in [version 1.53.0 of the Terraform AWS Provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1530-december-20-2018).

Can you confirm your provider is updated beyond that version?

**Comment by burtsdenis** [comment]
@bflad, hello!

We're using 2.2 version of Terraform AWS provider:
```
$ terraform providers
.
├── provider.aws ~> 2.2
```

> Hi @daniilyar 👋 Which Terraform AWS Provider version are you using? e.g.
> 
> ```
> $ terraform providers
> .
> └── provider.aws 2.1.0
> ```
> Support for `target_type = "lambda"` (along with validating values in that argument) was added via #6719 and released in [version 1.53.0 of the Terraform AWS Provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1530-december-20-2018).
> 
> Can you confirm your provider is updated beyond that version?

**Comment by daniilyar** [comment]
@bflad , yes, works for us as well with newer version of the provider. Closing the ticket, thanks!
